src/IOT/lab1/Main.java

```diff
@@ -0,0 +1,15 @@
+package IOT.lab1;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Crank1 crank1 = new Crank1();
+        Crank1 crank2 = new Crank1(700, "50*60*50", "mechanical");
```

Comment:
Are you sure you want your producer to be "50\*60\*50" (which should be a value for size field, I suppose) and "mechanical" as a color?